DataProtection.sln

```diff
@@ -33,6 +32,20 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.AspNetCore.DataPr
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.AspNetCore.DataProtection.Extensions", "src\Microsoft.AspNetCore.DataProtection.Extensions\Microsoft.AspNetCore.DataProtection.Extensions.xproj", "{BF8681DB-C28B-441F-BD92-0DCFE9537A9F}"
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.AspNetCore.DataProtection.Azure", "src\Microsoft.AspNetCore.DataProtection.Azure\Microsoft.AspNetCore.DataProtection.Azure.xproj", "{CC799B57-81E2-4F45-8A32-0D5F49753C3F}"
```

Comment:
Maybe it should be called something more specific? Like blob storage? Are we going to put all azure support things in here?